common/djangoapps/third_party_auth/urls.py

```diff
@@ -2,11 +2,12 @@
 
 from django.conf.urls import include, patterns, url
 
-from .views import inactive_user_view, saml_metadata_view
+from .views import inactive_user_view, saml_metadata_view, lti_login_view
 
 urlpatterns = patterns(
     '',
     url(r'^auth/inactive', inactive_user_view),
     url(r'^auth/saml/metadata.xml', saml_metadata_view),
+    url(r'^auth/login/(?P<backend>lti)/$', lti_login_view),
```

Comment:
I don't have a strong opinion on that. We can also consider renaming the method `lti_login_view`, even if the URL is kept as "login".